Makefile

```diff
@@ -61,7 +61,22 @@ METRICS_PORT ?= 9118
 # Preferred Capella version
 CAPELLA_VERSION ?= 5.2.0
 
-DOCKER_TAG ?= latest
+# Should be 'latest', the branch name, the commit hash or a Git tag name
+CAPELLA_DOCKERIMAGES_VERSION ?= latest
```

Comment:
I suggest to rename
`CAPELLA_DOCKERIMAGES_VERSION`
to
`CAPELLA_DOCKERIMAGES_REVISION`
since it is a revision information that is to be given here and not really a version in all cases.

Reply:
There is just one case where it is not the revision: `latest`. 
From my understanding, version is a general term. I'd say a commit hash is also version. In most of the cases it's a Git tag (semantic version) anyway. What do you think?

Reply:
The reason why I suggest to change that is that `CAPELLA_VERSION` points to some [semantic version](https://semver.org/). For `CAPELLA_DOCKERIMAGES_REVISION` it is different and that might be confusing for others. The comment in 64 describes a revision I would say. And yes, `latest` is a special case. Especially in the context of Docker images. But that's fine I'd say.

Reply:
Done